test/e2e/utils/common.go

```diff
@@ -173,10 +172,18 @@ func DefaultSpec(t *testing.T, testKube *TestKubernetes, initializer func(*ispnv
 	if initializer != nil {
 		initializer(infinispan)
 	}
+
+	if OperandVersion != "" {
+		infinispan.Spec.Version = OperandVersion
+	}
```

Comment:
If we add this after the `if RunLocalOperator == TRUE` block, then we can overwrite in both modes of operation.

Reply:
Good point